### PR TITLE
Clean up the renaming of collect to receive in drips

### PR DESCRIPTION
**Pull request by CodeSandwich**

### CODE REVIEW

File: src/Drips.sol [hunk 1]
```diff
@@ -69,18 +69,18 @@ library Drips {
     struct DripsState {
         /// @notice Drips receivers list hash, see `hashDrips`.
         bytes32 dripsHash;
-        /// @notice The next cycle to be collected
-        uint32 nextCollectedCycle;
+        /// @notice The next cycle to be received
+        uint32 nextReceivedCycle;
```
Comment (xmxanuel) [inline src/Drips.sol:73]: Maybe
```suggestion
        uint32 nextReceivableCycle;
```

Comment (xmxanuel) [reply src/Drips.sol:73]: Because the receive didn't happen yet

**Review by xmxanuel — APPROVED** [review]
lgtm

File: src/Drips.sol (hunk 1 above)
Comment (CodeSandwich) [reply src/Drips.sol]: Good point, fixed

**Pull request merged**